MoreLinq/Repeat.cs

```diff
@@ -23,7 +23,7 @@ namespace MoreLinq
     public static partial class MoreEnumerable
     {
         /// <summary>
-        /// Repeats the specific sequences <paramref name="count"/> times.
+        /// Repeats the sequence <paramref name="count"/> times.
```

Comment:
Summaries read stand-alone without any references like `<paramref>` here for `count`.